<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="msg_edit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>291</width>
     <height>171</height>
    </rect>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::NoWrap</enum>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="login_button">
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="password_button">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>315</width>
     <height>85</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="user_edit">
      <property name="minimumSize">
       <size>
        <width>194</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密  码：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="pwd_edit"/>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>label_2</zorder>
   <zorder>pwd_edit</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>label</zorder>
   <zorder>user_edit</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>login_button</sender>
   <signal>clicked()</signal>
   <receiver>msg_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
